.framework/node/backend/lib/event.js

```diff
@@ -12,8 +12,12 @@ const axios = axiosLib.create({
 });
 
 async function sendEvent(event, metadata) {
-  const result = await axios.post(`/users/${WILCO_ID}/event`, JSON.stringify({ event, metadata }));
-  return result.data;
+  try {
+    const result = await axios.post(`/users/${WILCO_ID}/event`, JSON.stringify({event, metadata}));
+    return result?.data;
+  } catch (e) {
+    // pass
+  }
```

Comment:
Why is the try-catch needed now?

Reply:
When the wilco_id is not found in the request to the engine, it returns 4XX, and express throws an error

Reply:
Can we add env variable to the worker with wilco_id to overcome it?

Reply:
OK got it